frontend/javascripts/oxalis/model/sagas/annotation_saga.tsx

```diff
@@ -47,12 +45,37 @@ import { determineLayout } from "oxalis/view/layouting/default_layout_configs";
 import { getLastActiveLayout, getLayoutConfig } from "oxalis/view/layouting/layout_persistence";
 import { is3dViewportMaximized } from "oxalis/view/layouting/flex_layout_helper";
 import { needsLocalHdf5Mapping } from "../accessors/volumetracing_accessor";
+import { pushSaveQueueTransaction } from "../actions/save_actions";
+import { updateAnnotationLayerName, updateMetadataOfAnnotation } from "./update_actions";
 
 /* Note that this must stay in sync with the back-end constant MaxMagForAgglomerateMapping
   compare https://github.com/scalableminds/webknossos/issues/5223.
  */
 const MAX_MAG_FOR_AGGLOMERATE_MAPPING = 16;
 
+export function* pushAnnotationNameUpdateAction(action: SetAnnotationNameAction) {
+  const mayEdit = yield* select((state) => mayEditAnnotationProperties(state));
+  if (!mayEdit) {
+    return;
+  }
+  yield* put(
+    pushSaveQueueTransaction([updateMetadataOfAnnotation(action.name)], "unused-tracing-id"),
```

Comment:
i find this `unused-tracing-id` parameter weird. not sure yet how we can avoid this best.

Reply:
I refactored this away by not adding the id automatically later, but making it required to pass it early on (if the specific action requires it).